src/utils/dates.js

```diff
@@ -1,5 +1,18 @@
 const ONE_DAY_IN_MILLISECONDS = 86400000;
 
+export function getDaysBetweenDates(date1, date2) {
+	// If either date is undefined, return 14 days. This happens when the user
+	// has just created an item and hasn't purchased it yet.
+	if (date1 === undefined || date2 === undefined) {
```

Comment:
Do we need to check that both dates could be undefined? Based on how this function is used, would `dateNextPurchased` ever be undefined?